docs/imviz/plugins.rst

```diff
@@ -87,12 +93,18 @@ an interactively selected region. A typical workflow is as follows:
 2. Draw a region over the object of interest (see :ref:`imviz_defining_spatial_regions`).
 3. Select the desired image using the :guilabel:`Data` drop-down menu.
 4. Select the desired region using the :guilabel:`Subset` drop-down menu.
-5. If you want to subtract background before performing photometry, enter
-   the background value in the :guilabel:`Background value` field.
-   This value must be in the same unit as display data, if applicable.
-   Alternately, you can also define a region for background calculation (median)
-   and select that region using the :guilabel:`Subset (background)` drop-down menu.
-   If your image is already background subtracted, leave it at 0.
+5. If you want to subtract background before performing photometry,
+   you have the following 3 options. Otherwise if your image is already
+   background subtracted, leave it at 0:
+
+  * Manual: Enter the background value in the :guilabel:`Background value` field.
+    This value must be in the same unit as display data, if applicable.
+  * Annulus: Enter its inner radius and width in the :guilabel:`Annulus inner radius`
+    and :guilabel:`Annulus width`, respectively. Median of the pixels within
+    the annulus region will be used but the annulus shape will not be shown on display.
```

Comment:
is the display feedback out-of-scope?  Is it planned for future development?  After trying to use this for testing, I think visual feedback would be extremely useful to ensure you're doing what you think you're doing - especially with rectangular apertures and crowded fields where you want to make sure the annulus does not include other sources, etc.

Reply:
Yes, POs specifically said do not display the annulus this round. Getting annulus to display, etc, is a lot of work because it involves new feature for glue-astronomy translator:

* glue-viz/glue-astronomy#49

Reply:
Also, this plugin is not for crowded field photometry, which would probably need PSF photometry that is out of scope.

Reply:
ok, another option (either now or later) would be just lines representing the inner and outer radii of the annulus.  I suspect that would be much easier?

Reply:
Re: Just lines -- I thought about that but the only API I can think of is the marker and I find the following unsolved issue too disruptive to include (in addition to all the linking and de-linking overhead that comes with adding marker table):

* #760

Reply:
If you know of a way to draw lines without using markers, it would be nice but I don't think it can fit into this sprint because it is out of scope.

Reply:
ok, I have an idea how to do it fairly quickly but probably won't have time left this week either.  I agree its not blocking for this PR, but do think it would make this work more usable so would like to see it prioritized soon since it shouldn't be a ton of effort.

[:cat:](https://jira.stsci.edu/browse/JDAT-2324)